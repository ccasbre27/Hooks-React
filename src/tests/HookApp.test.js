import React from 'react';
import { shallow } from "enzyme";
import { App } from '../App';

describe('Tests on Hook App', () => {
   
    test('should render the component', () => {
       
        const wrapper = shallow( <App /> );
        expect( wrapper ).toMatchSnapshot();

    });
    

});
