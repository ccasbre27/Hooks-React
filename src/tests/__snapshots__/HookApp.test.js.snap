// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests on Hook App should render the component 1`] = `
<div
  className="App"
/>
`;
